tsconfig.json

```diff
@@ -1,15 +1,11 @@
 {
-  "files": [
-    "./*.ts"
-  ],
   "compilerOptions": {
     "rootDir": "./",
-    "baseUrl": "./",
-    "target": "es2017", // Node.js 8
-    "module": "commonjs",
+    "outDir": "dist",
+    "alwaysStrict": false,
+    "target": "es5", // Node.js 8
```

Comment:
Node 8 is not maintained anymore, so feel free to bump up to a more modern target.

The build process currently uses babel to transpile and minify, so anyone who needs legacy support can use the minified version: `/dist/fetch-json.min.js`

See `transpileES6` at:
https://github.com/center-key/fetch-json/blob/master/gulpfile.js#L22

Any thoughts on a good way for **tsconfig.json** `"target"` and babel transpile to work together?  I'm not sure what the current best practice is for coordinating these related processes.

Reply:
👍 I was thinking there surely is a gulp typescript plugin that allows us to run the `ts` version through the regular gulp streams, so we even bypass the need for an intermediary `build` folder.

I know there's a `babel-typescript` preset so we can let babel do the transpilation directly on the ts file.

What do you think?

Reply:
Bypassing the intermediary step would be awesome.

Reply:
Hi @dpilafian! Sorry for the late reply, I am reworking these bits as discussed :)

Reply:
@dpilafian I think I solved it now in the latest commit. So gulp now handles everything and there's no more intermediary build step. Have a look and let me know what you think :)